test/functional/rpc_rawtransaction.py

```diff
@@ -339,7 +339,7 @@ def run_test(self):
 
         # 5. valid parameters - supply txid and True for non-verbose
         assert_equal(self.nodes[0].getrawtransaction(txId, True)["hex"], rawTxSigned['hex'])
-        assert_equal(self.nodes[0].getrawtransactionmulti({"0":[txId]}, True)[txId]['hex'], rawTxSigned['hex'])
+        assert_equal(self.nodes[0].getrawtransactionmulti(verbose=True, transactions={"0":[txId]})[txId]['hex'], rawTxSigned['hex'])
```

Comment:
0009c5e81eb0e07b930eea5bf40bf95ac260e20c: nit: why changing args order here?

Reply:
doesn't really matter, I just wanted to be sure that it is parsed by name not by its number.

Should I revert it?

Reply:
no, that's fine, I was just wondering if I maybe missed something here